components/swatchgroup/index.css

```diff
@@ -13,24 +13,27 @@
 
 .spectrum-SwatchGroup {
 	--spectrum-swatchgroup-spacing: var(--spectrum-spacing-75);
+}
 
+.spectrum-SwatchGroup {
 	display: inline-flex;
 	flex-flow: row wrap;
 	align-items: flex-start;
 	justify-content: flex-start;
+	gap: var(--mod-swatchgroup-spacing, var(--spectrum-swatchgroup-spacing));
+}
 
-	/* Regular (Default) */
-	gap: var(--mod-swatchgroup-spacing-regular, var(--spectrum-swatchgroup-spacing));
+/* Spacious */
+.spectrum-SwatchGroup--spacious {
+	--mod-swatchgroup-spacing: var(--mod-swatch-group-spacing-spacious, var(--spectrum-swatch-group-spacing-spacious));
 }
 
-/* Compact */
-.spectrum-SwatchGroup--compact {
-	--mod-swatchgroup-spacing-regular: var(--mod-swatchgroup-spacing-compact);
-	--spectrum-swatchgroup-spacing: var(--spectrum-spacing-50);
+/* Medium */
+.spectrum-SwatchGroup--sizeM {
+	--spectrum-swatchgroup-spacing: var(--spectrum-spacing-100);
 }
```

Comment:
What was our take away on using medium? If it's the default are we just rolling it into the root class now like SWC does or do we want to support it separately? I know we said we don't want to require it though so the root class needs this value defined too if both are present.